### PR TITLE
[CI] Install Codecov on Windows

### PR DESCRIPTION
**Pull request by amenasria**
This PR installs the [Codecov uploader](https://docs.codecov.com/v4.6/docs/codecov-uploader) on Windows.

### CODE REVIEW

File: windows/helpers/phase1/install_codecov.ps1 [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    if (-Not (Test-Path -Path $CodecovPath)) {
+        Write-Output "Creating $CodecovPath"
+        mkdir $CodecovPath
+    }
+    Write-Output "Moving codecov.exe to $CodecovPath"
+    Move-Item -Path $OutFile -Destination "$CodecovPath\codecov.exe"
+}
+
+
+# Adding codecov folder to the system PATH
```
Comment (KevinFairise2) [inline windows/helpers/phase1/install_codecov.ps1:26]: The comment is wrong here no? I think it only get the current PATH

Comment (amenasria) [reply windows/helpers/phase1/install_codecov.ps1:26]: The comment was meant to describe the 10 next lines

**Review by CelianR — APPROVED** [review]

**Review by iglendd — APPROVED** [review]
LGTM

File: windows/helpers/phase1/install_codecov.ps1 [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+$Path = [System.Environment]::GetEnvironmentVariable("PATH", "Machine")
+
+# Checking if codecov folder is already in the path
+if ($Path -notlike "*$CodecovPath*")
```
Comment (derekwbrown) [inline windows/helpers/phase1/install_codecov.ps1:30]: There is an `Add-ToPath()` helper which does much the same thing.
Will also allow you to add to the container path if this is needed later in the container creation

Comment (amenasria) [reply windows/helpers/phase1/install_codecov.ps1]: Didn't know about this helper, thanks Derek ! Fixed with [c290282](https://github.com/DataDog/datadog-agent-buildimages/pull/582/commits/c2902820549f063cf442d68b5f41eef08d5ffa71).

**Pull request merged**